### ISSUE TITLE
Надо скопировать авиакомпании из таблицы старой базы в новую базу

### ISSUE THREAD

**Issue by tsv19su** [tsv19su254052/LoadWorkDataAirFlightsDBNew#61]
Надо скопировать авиакомпании из таблицы старой базы в новую базу

_Originally posted by @tsv19su254052 in https://github.com/tsv19su254052/LoadWorkDataAirFlightsDBNew/issues/1#issuecomment-1163017376_

**Comment by tsv19su** [comment]
Согласен. Надо было сделать это с самого начала. Обработка если не находит авиакомпанию, вставляет пустышку с одним кодом IATA. А потом приходится вылавливать эти пустышки и заменять вручную на настоящие

**Comment by tsv19su254052** [comment]
Отлично.
Просто класс
